pulp_container/app/checks.py

```diff
@@ -0,0 +1,43 @@
+from django.conf import settings
+from django.core.checks import Error as CheckError, register
+
+
+@register(deploy=True)
+def container_settings_check(app_configs, **kwargs):
+    errors = []
+
+    # Other checks only apply if token auth is enabled
+    if str(getattr(settings, "TOKEN_AUTH_DISABLED", False)).lower() == "true":
+        return errors
+
+    if getattr(settings, "TOKEN_SERVER", None) is None:
+        errors.append(
+            CheckError(
+                "TOKEN_SERVER variable has to be set when token authentification is enabled"
+            ),
+            id="pulp_container.E001",
+        )
+    if getattr(settings, "TOKEN_SIGNATURE_ALGORITHM", None) is None:
+        errors.append(
+            CheckError(
+                "TOKEN_SIGNATURE_ALGORITHM variable has to be set when token authentification"
+                " is enabled",
+                id="pulp_container.E002",
+            )
+        )
+    if getattr(settings, "PUBLIC_KEY_PATH", None) is None:
+        errors.append(
+            CheckError(
+                "PUBLIC_KEY_PATH variable has to be set when token authentification is enabled",
+                id="pulp_container.E003",
+            )
+        )
+    if getattr(settings, "PRIVATE_KEY_PATH", None) is None:
+        errors.append(
+            CheckError(
+                "PRIVATE_KEY_PATH variable has to be set when token authentification is enabled",
+                id="pulp_container.E004",
+            )
+        )
```

Comment:
```suggestion
    if getattr(settings, "PRIVATE_KEY_PATH", None) is None:
        errors.append(
            CheckError(
                "PRIVATE_KEY_PATH variable has to be set when token authentification is enabled",
                id="pulp_container.E001",
            )
        )
```
I think we can use the same error ID. This opinion is not backed by anything.